lunch-go-where/src/pages/Landing.js

```diff
@@ -26,6 +26,7 @@ const style = {
   border: "2px solid #000",
```

Comment:
If you want you can consider removing this, do you think the modal looks a bit weird with a black border? Just a thought.

Reply:
Yes i think it looks better hehe thanksu! Made the changes :D